src/goto-instrument/contracts/contracts.cpp

```diff
@@ -1122,10 +1136,9 @@ bool code_contractst::check_frame_conditions_function(const irep_idt &function)
     return true;
   }
 
-  if(check_for_looped_mallocs(function_obj->second.body))
-  {
-    return true;
-  }
+  // unpack
```

Comment:
Unpack what? I think we can remove this comment.

Reply:
fixed